src/molecules/formfields/ToggleField/index.js

```diff
@@ -7,13 +7,13 @@ import Tooltip from 'atoms/Tooltip';
 import Button from 'atoms/Button';
 import Layout, { Col } from 'atoms/Layout';
 
-import styles from '../shared/formfields.module.scss';
+import styles from './toggle_field.module.scss';
 
 const renderChoices = (choices, input) => {
   if (!choices) return null;
 
   const renderChoice = (choice, idx) => {
-    const variantName = input.value === choice.value ? 'toggle-active' : 'toggle';
+    const variantName = input.value == choice.value ? 'toggle-active' : 'toggle'; // eslint-disable-line
```

Comment:
probably not: database might send a string, it might send a number, depends on the use case. the `input` will always store the value as a string. Probably need to use coercive equality to lessen code bloat